.github/workflows/vul.yml

```diff
@@ -22,13 +22,13 @@ on:
             - main
 
 jobs:
-  build_test:
+  build_and_test:
     runs-on: ${{matrix.os}}
     strategy:
       fail-fast: false
       matrix:
         config: [Debug, Release]
-        os: [ ubuntu-latest, windows-latest, macos-latest ]
+        os: [ ubuntu-18.04, ubuntu-20.04, ubuntu-22.04, windows-2019, windows-2022, macos-11, macos-12 ]
```

Comment:
There is so little ressource on Github actions, I am not sure it's a great idea to rely too much on this... we have to wait hours for the tests to complete...

This said, the library is likely not going to be updated too often.

Reply:
It's only lately that github actions have been a bit slow. But it's affecting everyone and should be fixed soon.